### PR TITLE
NS-231, kick off new Nessie job; required params in path, if necessary

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/NS-231

### CODE REVIEW

File: src/components/Jobs.vue [hunk 1]
```diff
@@ -92,7 +96,21 @@ export default {
       });
     },
     runSelectedJob() {
-      console.log('Run job: ' + this.runnableJob.selected.name);
+      let apiPath = this.runnableJob.selected.path;
```
Comment (paulkerschen) [inline src/components/Jobs.vue]: @johncrossman Idle curiosity - why `let` here? I know there are subtle effects on scoping but am not used to the idioms.

File: src/components/Jobs.vue [hunk 2]
```diff
@@ -92,7 +96,21 @@ export default {
...
+      this._.each(
+        this.runnableJob.selected.requiredParameters,
+        (requiredParameter, index) => {
+          apiPath = this._.replace(
```
Comment (paulkerschen) [inline src/components/Jobs.vue]: So `this._` is the usual way to inject Lodash into Vue?

Comment (johncrossman) [reply src/components/Jobs.vue]: Yes, a big odd – see https://github.com/Ewocker/vue-lodash  There might be a better util and/or usage.

File: src/components/Jobs.vue (hunk 1 above)
Comment (johncrossman) [reply src/components/Jobs.vue]: `let` is preferred ES6 usage. See https://vegibit.com/es6-let-vs-var-vs-const/

**Comment by paulkerschen** [comment]
Thanks @johncrossman!

**Pull request merged**